webapp/platform/client/src/client4.ts

```diff
@@ -2443,6 +2443,7 @@ export default class Client4 {
             ActiveSearchBackend: string;
             database_status: string;
             filestore_status: string;
+            root_status: string;
```

Comment:
Shouldn't this be a `boolean`?

Reply:
Since the API sends `root_status` as a `string`, a string is accepted here.

```
{
    "ActiveSearchBackend": "database",
    "AndroidLatestVersion": "",
    "AndroidMinVersion": "",
    "IosLatestVersion": "",
    "IosMinVersion": "",
    "database_status": "OK",
    "filestore_status": "OK",
    "root_status": "false",
    "status": "OK"
}
```
I decided to leave it as a string since I felt it would be simpler that way.
I could change [this](https://github.com/mattermost/mattermost/pull/26130/files#diff-14c336b2bbe04dd401717a01b70735320047916245e31c145af13bb21a47a869R136) to `s := make(map[string]interface{})` and pass the `root_status` as a `boolean`.

Do you want me to change it to `boolean` instead?

Reply:
Yeah, I think we should change the map to  a `map[string]any`. :+1: